pynucastro/rates/tests/test_tabular_library.py

```diff
@@ -45,6 +49,10 @@ def test_ordering(self):
         oda_rates = [r for r in tl_new.get_rates() if r.rfile.startswith("oda")]
         assert len(oda_rates) == 20
 
```

Comment:
I think this would be a better test if pruet was included in the ordering, unless there's a reason it can't be

Reply:
the point of this test was just to show that swapping FFN to have higher precedence changes things.  No sources overlap with the Pruet rates, so its place in the ordering doesn't matter.